packages/std-client/src/setup/setupMUDNetwork.ts

```diff
@@ -155,7 +155,8 @@ export async function setupMUDNetwork<C extends ContractComponents, SystemTypes
         provider: providerConfig,
         worldContract: contractsConfig.World,
         initialBlockNumber: networkConfig.initialBlockNumber ?? 0,
-        disableCache: networkConfig.devMode, // Disable cache on local networks (hardhat / anvil)
+        // TODO: `networkConfig.devMode` is deprecated, only use `networkConfig.disableCache`
+        disableCache: networkConfig.devMode || networkConfig.disableCache,
```

Comment:
I'd be fine just removing dev mode!

Reply:
unfortunately removing it would be a breaking change if we ever want to do another patch release for v1. I removed it in `setupMUD2Network` though.